File: backend/src/main/java/tom/projects/monkeylog/repository/measurement/MeasurementRepository.java
package tom.projects.monkeylog.repository.measurement;

import org.springframework.data.jpa.repository.JpaRepository;
import tom.projects.monkeylog.model.measurement.Measurement;

import java.util.UUID;

public interface MeasurementRepository extends JpaRepository<Measurement, UUID> {
}

File: backend/bin/main/tom/com/monkeylog/repository/measurement/MeasurementRepository.kt
package tom.com.monkeylog.repository.measurement

import org.springframework.data.jpa.repository.EntityGraph
import org.springframework.data.jpa.repository.JpaRepository
import tom.com.monkeylog.model.measurement.Measurement
import java.util.*

interface MeasurementRepository : JpaRepository<Measurement, UUID> {
    @EntityGraph(attributePaths = ["measurementPoints"])
    override fun findAll(): List<Measurement>
}
